Samples.xcodeproj/project.pbxproj

```diff
@@ -556,6 +559,12 @@
 				218F35B329C28F4A00502022 /* AuthenticateWithOAuthView.swift */,
 			);
 			path = "Authenticate with OAuth";
+		1C08854D29CA400500BA091A /* Change viewpoint */ = {
```

Comment:
It is because of the structural data is messed up in the project. When you manually edit the project file (during a conflict resolve), make sure that the edited part look correct comparing to the existing ones.